src/Form/DeliveryType.php

```diff
@@ -149,8 +165,8 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ]);
             }
 
-            $isDeliveryOrder = null !== $store && $store->getCreateOrders();
-            
+            $isDeliveryOrder = null !== $store;
```

Comment:
Updated the check. There is a way to reach this form for food tech orders too, so I removed `Save` button for food tech orders in the last commit, so that nobody edits it by mistake

Reply:
As I see it, we have three types of orders:

- food tech ("shop"/restaurant)
- local commerce and last mile ("store")
- B2C via Order form

The difference is that only for "stores" does it make sense to edit a delivery, set arbitrary price, recurrence rules etc. For both food tech and B2C deliveries, it's a one-off order that is already paid so the page is read-only for now

Reply:
yes indeed I agree - maybe add this message as a comment ?

ah yes actually you could access past foodtech deliveries from the "deliveries" panel and then access this form?